profile/README.md

```diff
@@ -19,4 +19,4 @@ If you would like to be more up to date on the progress within Pulsar feel free
 * [Frequently Asked Questions](https://github.com/pulsar-edit/.github/blob/main/FAQ.md)
 * [Contributing Right Now](https://github.com/pulsar-edit/.github/blob/main/project-birth/CONTRIBUTING-DURING-START.md)
 
-Otherwise you may always check out our [Discord Server](https://discord.gg/gp9UStzsHk).
+Otherwise you may always check out our [Discord Server](https://discord.gg/gp9UStzsHk) or [Subreddit](https://www.reddit.com/r/pulsaredit/).
```

Comment:
```suggestion
Otherwise you may always check out our [Discord Server](https://discord.gg/gp9UStzsHk).
```

Since these changes have been accomplished in PR #15

Reply:
I would close this one

Reply:
No i wouldnt

Reply:
I’ll leave it linked in the contact section